opta/core/terraform.py

```diff
@@ -351,8 +347,13 @@ def download_state(cls, layer: "Layer") -> bool:
         else:
             raise UserErrors("Need to get state from S3 or GCS")
 
-        cls.downloaded_state[layer.name] = True
-        return True
+        with open(state_file, "r") as file:
+            raw_state = file.read().replace("\n", "")
```

Comment:
why do we need this? yaml is whitespace sensitive - so won't it get messed up by this?

Reply:
`if raw_state != ""`